### PR TITLE
Split start conversation capability

### LINKED ISSUE #8513: Allow to limit who can create public and open conversations
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

**Is your feature request related to a problem? Please describe.**
In some situations (schools) it might be okay for users to be able to create group conversations but not public/open conversations.

Currently there is only a single setting:
- Limit creating a public and group conversation

We would need to split this into 2:
- Limit creating group conversations (basically the current one)
- Limit creating public and open conversations (new, but should copy config value from above on migration)

### PR DESCRIPTION
**Pull request by vitormattos**
### 🚧 TODO

- [ ] Close #8513

### 🏁 Checklist

- [x] ⛑️ Tests (unit and/or integration) are included
- [x] 📘 API documentation in `docs/` has been updated or is not required
- [x] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required
- [x] 🔖 Capability is added or not needed

### CODE REVIEW

File: docs/capabilities.md [hunk 1]
```diff
@@ -112,3 +112,5 @@ title: Capabilities
 * `recording-v1` - Call recording is available.
 * `config => call => breakout-rooms` - Whether breakout rooms are enabled on this instance
 * `config => call => recording` - Whether calls can be recorded (requires the High-performance backend server)
+* `config => conversations => can-create-group` - Whether the user can create group conversations, if not only one-to-one conversations are allowed
+* `config => conversations => can-create-public` - Whether the user can create public conversations, if not only one-to-one conversations are allowed
```
Comment (nickvergessen) [inline docs/capabilities.md:116]: But maybe the user can create group + 1-1?

**Comment by nickvergessen** [comment]
When a user can not create a public/group conversation, they should also not be able to make an "Open conversation" which would allow others to join it too?

**Comment by nickvergessen** [comment]
> Do you want to disallow to create 1-1 groups when haven't permission to create public conversation?

No, that's exactly the difference: 1-1 is not **public**

**Comment by vitormattos** [comment]
Rebased with master.

**Pull request closed without merging**